packages/thirdweb/src/extensions/erc1155/read/getNFTs.ts

```diff
@@ -84,8 +84,14 @@ async function getNFTsFromInsight(
     Math.max(0, Number(supply) - currentOffset),
   );
   if (result.length < expectedResultLength) {
-    // fresh contracts might be delayed in indexing, so we fallback to RPC
-    return getNFTsFromRPC(options);
+    try {
+      // fresh contracts might be delayed in indexing, so we fallback to RPC
+      const rpcResult = await getNFTsFromRPC(options);
+      return rpcResult;
```

Comment:
just return it directly? why the re-assignment?

Reply:
`return await getNFTsFromRPC(options);` ?

Reply:
looks a bit weird, but sure

Reply:
updated